setup.cfg

```diff
@@ -1,3 +1,3 @@
 [pycodestyle]
-ignore = E402
-max-line-length = 80
+ignore = E121, E123, E126, E226, E24, E704, W503, E402
+max-line-length = 120
```

Comment:
I don't agree. 80 makes projects less readable because of forced line breaks and is usually used for legacy reasons. PyCharm also uses 120 by default and it's a nice balance between being able to code on a potato and on a widescreen.